test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1776,6 +1776,14 @@ items:
     - ""
     attributeRestrictions: null
     resources:
+    - nodes
```

Comment:
> Its also required. If we expect the F5 to participate in the SDN it needs to know about the nodes its talking. We don't have OpenShift-sdn doing that. If this is a big problem it would mean splitting the f5 into its own role, I guess?

Or adding it as the delta required.